src/main/scala/com.snowplowanalytics/forex/OerClient.scala

```diff
@@ -182,25 +181,25 @@ case class OerClient[F[_]: Sync](
                   usdOverCurr.bigDecimal
                 )
               )
-          })
+          }
         // For Enterprise and Unlimited users, OER allows them to configure the base currencies.
         // So the exchange rate returned from the API is between target currency and the base
         // currency they defined.
         case _ =>
-          Sync[F].delay(response.rates.foreach {
+          response.rates.toList.traverse_ {
             case (currentCurrency, currencyValue) =>
               val keyPair = (config.baseCurrency, currentCurrency, date)
```

Comment:
Looks more like a `keyTriple`?

Reply:
This name is carried over from 0.6.0. While I agree it's misleading, the proper solution might actually be to address the fact that we have two caches that store the exact same types of data but distribute the datapoints differently across key and value.